src/plugins/aws/ssm/index.ts

```diff
@@ -148,29 +147,11 @@ const createPortForwardingCommand = (
   ];
 };
 
-type SsmCommands = { shellCommand: string[]; subCommand?: string[] };
+type SubCommand = string[];
 
-const createSsmCommands = (args: Omit<SsmArgs, "requestId">): SsmCommands => {
-  const interactiveShellCommand = createInteractiveShellCommand(args);
-
-  const forwardPortAddress = args.forwardPortAddress;
-  if (!forwardPortAddress) {
-    return { shellCommand: interactiveShellCommand };
-  }
-
-  const portForwardingCommand = createPortForwardingCommand({
-    ...args,
-    forwardPortAddress,
-  });
-
-  if (args.noRemoteCommands) {
-    return { shellCommand: portForwardingCommand };
-  }
-
-  return {
-    shellCommand: interactiveShellCommand,
-    subCommand: portForwardingCommand,
-  };
```

Comment:
Moved to `ssh.ts`.
This `index.ts` file now contains functions used by both ssh and scp.